src/environmentd/src/bin/environmentd/main.rs

```diff
@@ -295,10 +295,14 @@ pub struct Args {
         default_value = "2100"
     )]
     orchestrator_process_base_service_port: u16,
-    /// Where the process orchestrator should store its metadata.
+    /// Where the process orchestrator should store secrets.
+    // TODO(benesch): the name "data directory" is historical. We should rename
+    // to "--orchestrator-process-secrets-directory", but there are some
+    // irritating backwards compatibility concerns to sort out with the upgrade
+    // test.
     #[clap(
         long,
-        env = "ORCHESTRATOR_PROCESSDATA_DIRECTORY",
+        env = "ORCHESTRATOR_PROCESS_DATA_DIRECTORY",
```

Comment:
This variable was used with its old name in mzcompose/services.py . I guess, to be safe, services.py needs to set the variable with both its old and new spelling.